birdhouse/default.env

```diff
@@ -52,3 +52,5 @@ export DEFAULT_CONF_DIRS='
 
 export USER_WORKSPACE_UID=1000
 export USER_WORKSPACE_GID=1000
+
+export WPS_OUTPUTS_DIR="${DATA_PERSIST_ROOT}/wps_outputs"
```

Comment:
Use single-quote for delayed eval and add to `DELAY_EVAL` list since it depends on another var.

Reply:
Seems to be fixed in a new [commit](https://github.com/bird-house/birdhouse-deploy/pull/356/commits/601749ea29663e69038542903f48f919a0ba48ba) from PR [#356](https://github.com/bird-house/birdhouse-deploy/pull/356), so I won't add it here for now, unless we decide to change things about the data volume concerns.

Reply:
Yes, ignore this comment, I saw the fix in PR https://github.com/bird-house/birdhouse-deploy/pull/356 which is already merged.